backend/controllers/image.js

```diff
@@ -1,81 +1,99 @@
-      if (!image) { // if the image does not exist
-        return res.status(404).json({ error: 'Image not found!' }); // return a 404 Not Found error
+    .then((image) => {
+      // call the then method, which adds a callback function to the promise, to handle the success case
+      if (!image) {
+        // if the image does not exist
+        return res.status(404).json({ error: "Image not found!" }); // return a 404 Not Found error
       } // end of if (!image) { ... }
-      if (image.userId !== req.auth.userId) { // if the userId property of the Image object does not match the userId property of the request body
-        return res.status(401).json({ error: 'Unauthorized request!' }); // return a 401 Unauthorized error
+      if (image.userId !== req.auth.userId) {
+        // if the userId property of the Image object does not match the userId property of the request body
+        return res.status(401).json({ error: "Unauthorized request!" }); // return a 401 Unauthorized error
       } // end of if (image.userId !== req.auth.userId) { ... }
-      const filename = image.ImageUrl.split('/Images/')[1]; // set the filename constant to the second element of the array returned by splitting the ImageUrl property of the Image object
-      fs.unlink('Images/' + filename, () => { // call the unlink method, which deletes the file with the specified path
+      const filename = image.ImageUrl.split("/Images/")[1]; // set the filename constant to the second element of the array returned by splitting the ImageUrl property of the Image object
+      fs.unlink("Images/" + filename, () => {
+        // call the unlink method, which deletes the file with the specified path
         Image.deleteOne({ _id: req.params.id }) // call the deleteOne method, which returns a promise, which resolves to the Image object with the specified id
-          .then(() => res.status(200).json({ message: 'Deleted!' })) // call the then method, which adds a callback function to the promise, to handle the success case
-          .catch(error => res.status(400).json({
-            error: error
-          })) // call the catch method, which adds a callback function to the promise, to handle the failure case
-        }); // end of .then(image => { ... })
-    })
+          .then(() => res.status(200).json({ message: "Deleted!" })) // call the then method, which adds a callback function to the promise, to handle the success case
+          .catch((error) =>
+            res.status(400).json({
+              error: error,
+            }),
+          ); // call the catch method, which adds a callback function to the promise, to handle the failure case
+      }); // end of .then(image => { ... })
+    });
 };
 
-exports.getAllStuff = (req, res) => { // call the get method, which adds a route to the router object, to handle GET requests to the /api/stuff endpoint
+exports.getAllStuff = (req, res) => {
```

Comment:
It would be lovely to get rid of all references to Stuff. The comment here is not correct.